docs/specification.md

```diff
@@ -240,10 +261,54 @@ wqet7IWbw2j2lmGuoKCaFlYS_R7szczz
 ```
 
 
+#### Terminology
 
----
+Because the encoding algorithm is recursive, this leads to a few different ways to represent a sequence collection. We refer to these representations in "levels". The level number represents the number of "lookups" you'd have to do from the "top level" digest. So, we have:
 
-### 2. API: A server RESTful API specification for retrieving and comparing sequence collections.
+##### Level 0 (AKA "top level")
+
+Just a plain digest. This corresponds to **0 database lookups**. Example:
+```
+a6748aa0f6a1e165f871dbed5e54ba62
+```
+
+##### Level 1
+
+What you'd get when you look up the digest with **1 database lookup** (no recursion). We sometimes refer to this as the "array digests" or "attribute digests", because it is made up a digest for each attribute in the sequence collection. Example:
+```
+{
+  "lengths": "4925cdbd780a71e332d13145141863c1",
+  "names": "ce04be1226e56f48da55b6c130d45b94",
+  "sequences": "3b379221b4d6ea26da26cec571e5911c"
+}
+```
+
+##### Level 2
+
+What you'd get with **2 database lookups** (1 recursive call). This is the most common representation, and hence, it the level of the *canonical seqcol representation*. Example:
```

Comment:
```suggestion
What you'd get with **2 database lookups**. This is the most common representation, and hence, it is the level of the *canonical seqcol representation*. Example:
```

Reply:
I see no point in complicating the matter by adding the number of recursive calls

Reply:
Here, the term "canonical seqcol representation" is used instead of the "canonical seqcol object representation". See my main review comments on this.